### DISCUSSION TITLE
Change the default language of GrapeJS to Portuguese (pt).

### DISCUSSION THREAD

**Discussion by PrAnacleto** [GrapesJS/grapesjs#5231] [Q&A]
**I'm using the Latest GrapeJS Version**

I would like to know, how can I change the default language of GrapesJS ? 
I Try this but didn't work
![Screenshot 2023-07-10 175512](https://github.com/GrapesJS/grapesjs/assets/50595168/bb9dc5fe-75b1-400d-a28d-c94efbef33c7)
![Screenshot 2023-07-10 175550](https://github.com/GrapesJS/grapesjs/assets/50595168/6e901743-346e-4160-b29b-b8f69b79ec2c)
Please help me

**Comment by artf** [comment]
You have to use the language exported object in GrapesJS configuration https://grapesjs.com/docs/modules/I18n.html#configuration
